### DISCUSSION TITLE
OCSP Verification

### DISCUSSION THREAD

**Discussion by BenjPs** [go-resty/resty#530] [Q&A] [answered]
Is this library able to do OCSP verification and if not, are there any plans for implementation in the future?

**Comment by moorereason** [comment, marked as answer]
No, resty does not provide that.  I'm not aware of any plans to add it.

However, you should be able to use a custom TLS Config with `client.SetTLSClientConfig` that implements OCSP verification.  I've not done OCSP verification before, but [this article](https://www.cossacklabs.com/blog/tls-validation-implementing-ocsp-and-crl-in-go/) seems to explain how to do it.

We may be able to incorporate that into resty later if we can agree to an API.
